[CI] Increase the shard numbers for torchbench tests

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #113668
* #113559
* #113446
* __->__ #113575
* #113574

Summary: torchbench tests are always the lagging shards when comparing to other integration test shards, so let's bump up the corresponding shard numbers.

desertfire committed:
[CI] Increase the shard numbers for torchbench tests

Summary: torchbench tests are always the lagging shards when comparing to other integration test shards, so let's bump up the corresponding shard numbers.

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Increase the shard numbers for torchbench tests"


Summary: torchbench tests are always the lagging shards when comparing to other integration test shards, so let's bump up the corresponding shard numbers.

[ghstack-poisoned]

ezyang reviewed (approved):
Can you check that the shards are actually well balanced, they are often not

desertfire committed:
Update on "[CI] Increase the shard numbers for torchbench tests"


Summary: torchbench tests are always the lagging shards when comparing to other integration test shards, so let's bump up the corresponding shard numbers.

[ghstack-poisoned]

desertfire commented:
> Can you check that the shards are actually well balanced, they are often not

I checked the timing in https://github.com/pytorch/pytorch/actions/runs/6877874087/job/18707772629. The newly split two torchbench shards are well balanced with similar execution time. For shards from other tests, some are larger and some are smaller, but they roughly all finish within 2 hours.

malfet commented:
> Can you check that the shards are actually well balanced, they are often not

If shard name have not change, previous runtime information will be used to determine what test will be run in which shard

desertfire committed:
Update on "[CI] Increase the shard numbers for torchbench tests"


Summary: torchbench tests are always the lagging shards when comparing to other integration test shards, so let's bump up the corresponding shard numbers.

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Increase the shard numbers for torchbench tests"


Summary: torchbench tests are always the lagging shards when comparing to other integration test shards, so let's bump up the corresponding shard numbers.

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Increase the shard numbers for torchbench tests"


Summary: torchbench tests are always the lagging shards when comparing to other integration test shards, so let's bump up the corresponding shard numbers.

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Increase the shard numbers for torchbench tests"


Summary: torchbench tests are always the lagging shards when comparing to other integration test shards, so let's bump up the corresponding shard numbers.

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Increase the shard numbers for torchbench tests"


Summary: torchbench tests are always the lagging shards when comparing to other integration test shards, so let's bump up the corresponding shard numbers.

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Increase the shard numbers for torchbench tests"


Summary: torchbench tests are always the lagging shards when comparing to other integration test shards, so let's bump up the corresponding shard numbers.

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Increase the shard numbers for torchbench tests"


Summary: torchbench tests are always the lagging shards when comparing to other integration test shards, so let's bump up the corresponding shard numbers.

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Increase the shard numbers for torchbench tests"


Summary: torchbench tests are always the lagging shards when comparing to other integration test shards, so let's bump up the corresponding shard numbers.

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Increase the shard numbers for torchbench tests"


Summary: torchbench tests are always the lagging shards when comparing to other integration test shards, so let's bump up the corresponding shard numbers.

[ghstack-poisoned]
